python_modules/dagster-graphql/dagster_graphql/schema/errors.py

```diff
@@ -495,6 +495,24 @@ def __init__(self, message=None):
         self.message = message if message else "Authorization failed"
 
 
+class GrapheneDuplicateDynamicPartitionError(graphene.ObjectType):
+    class Meta:
+        interfaces = (GrapheneError,)
+        name = "DuplicateDynamicPartitionError"
+
+    partitions_def_name = graphene.NonNull(graphene.String)
+    partition_name = graphene.NonNull(graphene.String)
+
+    def __init__(self, partitions_def_name, partition_name):
+        super().__init__()
+        self.partitions_def_name = check.str_param(partitions_def_name, "partitions_def_name")
+        self.partition_name = check.str_param(partition_name, "partition_name")
+        self.message = (
+            f"Partition {self.partition_name} already exists in dynamic partitions definition"
+            f" {self.partitions_def_name}."
+        )
+
+
 types = [
```

Comment:
Does it need to be included in this array

Reply:
most likely--i've added it now